zzzz_modular_occulus/code/modules/reagents/reagent_containers/hypovial.dm

```diff
@@ -76,61 +78,73 @@
 /obj/item/weapon/reagent_containers/glass/beaker/hypovial/small/bicaridine
 	name = "red hypovial (bicaridine)"
 	icon_state = "hypovial-b"
+	bad_type = /obj/item/weapon/reagent_containers/glass/beaker/hypovial/small/bicaridine
```

Comment:
For future reference, bad_type should be used to blacklist a prototype parent typepath (i.e. the one abstract item you make to set up all your vars before making children of it with actual stats and sprites and function) without affecting its children rather than for blacklisting individual items. Use `spawn_blacklisted = TRUE` for the latter.

Reply:
okay!